### PR TITLE
Access Correct Data Servlet

### PR DESCRIPTION
**Pull request by mbouadeus**
This pull request addresses a problem we were facing with accessing our data servlet for the different activity types: games, videos, and articles. Now when an activity type is selected, the appropriate data is retrieved from the backend.

Please disregard commits:
Change access modifiers.
Complete schedule activity page UI.
Add state to schedule activty page.
Add title field for event.
Merge branch 'master' of https://github.com/googleinterns/step170-2020 … …
Merge branch 'master' of https://github.com/googleinterns/step170-2020 … …
Merge branch 'mvp-dev' of https://github.com/googleinterns/step170-2020… …
Moved login state to app component.
Minor
Merge branch 'mvp-dev' of https://github.com/googleinterns/step170-2020… …
activities updated
Merge branch 'mvp-dev' of https://github.com/googleinterns/step170-2020… …
Move activity links to app component.
Merge branch 'mvp-dev' of https://github.com/googleinterns/step170-2020… …

They are handled in different pull requests.

### CODE REVIEW

**Review by trinitydonohugh — APPROVED** [review]

File: src/main/app/src/Routes.js [hunk 1]
```diff
@@ -1,30 +1,29 @@
...
+import HelpPage from './pages/HelpPage';
+import AboutPage from './pages/AboutPage';
+
+/* Routes address bar to corresponding page components */
+const Routes = ({activity, updateActivity, updateServlet, links}) => {
+    return (
+      <Switch>
+        <Route exact path='/' render={() => <HomePage activity={activity} updateActivity={updateActivity} updateServlet={updateServlet}/>}/>
+        <Route exact path='/schedule-activity' render={() => <ScheduleActivityPage activity={activity} updateActivity={updateActivity}/>}/>
+        <Route exact path ='/browse' render={() => <BrowsePage activity={activity} updateActivity={updateActivity} updateServlet={updateServlet} links={links} />}/>
```
Comment (pengxianzhe) [inline src/main/app/src/Routes.js]: there is a space between exact path and =, not sure if it breaks your code.

File: src/main/app/src/constants/Navbar.js [hunk 2]
```diff
@@ -19,12 +18,18 @@ const Navbar = () => {
   const handleLogin = (res) => {
     updateIsLoggedIn(true);
     updateGreeting("Welcome " + res.Ot.Cd +"!");
+    console.log(res);
```
Comment (pengxianzhe) [inline src/main/app/src/constants/Navbar.js]: Remove test code.

File: src/main/app/src/pages/HomePage.js [hunk 3]
```diff
@@ -4,15 +4,21 @@ import { Link } from 'react-router-dom';
 import 'bulma/css/bulma.css';
 import '../css/home.css';
 
+// Web Servlet links.
```
Comment (pengxianzhe) [inline src/main/app/src/pages/HomePage.js]: Is there a way to store these web servlet links so they can be shared across classes?

File: src/main/app/src/Routes.js (hunk 1 above)
Comment (mbouadeus) [reply src/main/app/src/Routes.js]: It doesn't but will change.

File: src/main/app/src/pages/HomePage.js (hunk 3 above)
Comment (mbouadeus) [reply src/main/app/src/pages/HomePage.js]: We moved it to App.js so all components can have access to it in a later pull request.

File: src/main/app/src/Routes.js (hunk 1 above)
Comment (mbouadeus) [reply src/main/app/src/Routes.js]: Fixed it.

File: src/main/app/src/constants/Navbar.js (hunk 2 above)
Comment (mbouadeus) [reply src/main/app/src/constants/Navbar.js]: Fixed it.

**Review by pengxianzhe — APPROVED** [review]

**Pull request merged**